packages/slate-editor/src/extensions/placeholders/PlaceholdersExtension.tsx

```diff
@@ -325,6 +339,20 @@ export function PlaceholdersExtension({
                 );
             }
             if (withVideoPlaceholders && isPlaceholderNode(element, PlaceholderNode.Type.VIDEO)) {
+                if (element.provider === PlaceholderNode.Provider.YOUTUBE) {
+                    return (
+                        <VideoPlaceholderElement
+                            attributes={attributes}
+                            element={element}
+                            fetchOembed={withVideoPlaceholders.fetchOembed}
+                            format={format}
+                            removable={removable}
+                        >
+                            {children}
+                        </VideoPlaceholderElement>
```

Comment:
I know that the content of these placeholders will change, for now I'm just using regular video and embed placeholder elements until design is ready.